pro_tes/exceptions.py

```diff
@@ -94,4 +106,16 @@ class InvalidMiddleware(MiddlewareException):
         "message": "Middleware is invalid.",
         "code": "500",
     },
+    InputUriError: {
+        "message": "Input URI cannot be parsed.",
+        "code": "400",
```

Comment:
Please sort the exceptions in this dictionary by the error code, in increasing order. The only exception is the very first exception `Exception` - please leave there where it is.